package.json

```diff
@@ -1272,8 +1276,8 @@
                     "group": "z_aws@2"
                 },
                 {
-                    "command": "aws.uploadLambda",
-                    "when": "explorerResourceIsFolder || isFileSystemResource && resourceFilename =~ /^template\\.(json|yml|yaml)$/",
+                    "command": "aws.uploadToAws",
```

Comment:
For now, let's leave `aws.uploadLambda` as it was, and add a new item for `aws.uploadToAws` that only works on folders.